site/content/docs/authentication/finding-a-verification-url.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Finding a Verification URL"
+tags: 
+- authentication
+- guide
+type: page
+---
+
+If you need to set up ZAP to handle authentication then you really need to find a suitable verification URL in your app.
+
+The verification URL will be one that you can request from the ZAP Manual Request Editor and which will send a response that allows you to work out whether you are logged in or not. What that looks like will entirely depend on your app, but could be something like:
+
+* Any page or API endpoint which returns the authenticated  user's Id
```

Comment:
There's still a double space here before "user's"

Reply:
Not any more - fixed in the last push ;)